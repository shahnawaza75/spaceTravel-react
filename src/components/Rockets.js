import React from 'react';
import { useSelector } from 'react-redux';
import RocketCard from './RocketCard';

const Rockets = () => {
  const rockets = useSelector((state) => state.rocket);

  return (
    <div>
      {rockets.map((item) => (
        <RocketCard
          key={item.id}
          data={item}
        />
      ))}
    </div>
  );
};
export default Rockets;
