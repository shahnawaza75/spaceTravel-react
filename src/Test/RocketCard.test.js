import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/index';
import Rockets from '../components/Rockets';

describe('Test the navigator', () => {
  it('Navigation is rendered', () => {
    const rockets = renderer
      .create(
        <Provider store={store}>
          <BrowserRouter>
            <Rockets />
          </BrowserRouter>
        </Provider>,
      )
      .toJSON();
    expect(rockets).toMatchSnapshot();
  });
});
