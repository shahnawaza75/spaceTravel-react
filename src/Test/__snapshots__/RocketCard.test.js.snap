// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the navigator Navigation is rendered 1`] = `<div />`;
